src/components/AddTodoForm/AddTodoForm.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import users from '../../api/users';
+import './AddTodoForm.css';
+
+export class AddTodoForm extends React.Component {
+  state = {
+    newTodoId: 0,
+    newTitle: '',
+    hasTitleError: false,
+
+    newUserId: 0,
+    hasUserError: false,
+  }
+
+  handleTitleChange = (event) => {
+    this.setState({
+      newTitle: event.target.value,
+      hasTitleError: false,
+      newTodoId: this.props.lengthForId + 1,
```

Comment:
I think would be better to increase the id when we submitting the form

Reply:
thanks